lib/deploy/handler.go

```diff
@@ -45,6 +47,25 @@ func Handler(ctx *gin.Context) {
 
 	fs.RemoveLocalRepo(path)
 
+	redisCtx := context.Background()
+
+	cmd := config.RedisClient.LPush(redisCtx, "builds", deployId.String())
+	if cmd.Err() != nil {
+		ctx.JSON(500, gin.H{
+			"error": cmd.Err().Error(),
+		})
+		return
+	}
```

Comment:
creates (if not exists) an array record called `builds` and pushes the build to there.
```
["3834e721-78a7-4e05-85ea-91b14895b70c", "bcc9dac6-4db3-4595-8918-db4fa1029cfb"]
```